.appveyor.yml

```diff
@@ -6,9 +6,9 @@ environment:
     - RUBY_VERSION: "26-x64"
 
 install:
-  - appveyor DownloadFile https://ftp.icm.edu.pl/pub/graphics/ImageMagick/binaries/ImageMagick-6.8.9-10-Q16-x64-dll.exe
-  - ImageMagick-6.8.9-10-Q16-x64-dll.exe /VERYSILENT /TASKS=install_Devel
-  - SET PATH=C:\Ruby%RUBY_VERSION%\bin;C:\Program Files\ImageMagick-6.8.9-Q16;%PATH%
+  - appveyor DownloadFile https://ftp.icm.edu.pl/pub/graphics/ImageMagick/binaries/ImageMagick-6.9.10-35-Q16-x64-dll.exe
```

Comment:
I can't agree this changing. I think we should add IM 6.7 and 6.9 instead.
it should test and guarantee to work with the IM versions that we supported on each platform.

Reply:
I have changed the AppVeyor build script to build those versions.